app/models/manageiq/providers/amazon/cloud_manager/refresher.rb

```diff
@@ -35,6 +44,29 @@ def parse_targeted_inventory(ems, _target, inventory)
     hashes
   end
 
+  def preprocess_targets
+    @targets_by_ems_id.each do |ems_id, targets|
+      if targets.any? { |t| t.kind_of?(ExtManagementSystem) }
+        ems             = @ems_by_ems_id[ems_id]
+        targets_for_log = targets.map { |t| "#{t.class} [#{t.name}] id [#{t.id}] " }
+        _log.info "Defaulting to full refresh for EMS: [#{ems.name}], id: [#{ems.id}], from targets: #{targets_for_log}" if targets.length > 1
+      end
+
+      # We want all targets of class EmsEvent to be merged into one target, so they can be refreshed together, otherwise
```

Comment:
So, this thing about `EmsEvent` being a target. Is https://github.com/ManageIQ/manageiq/pull/13408 needed for this to work?